src/renderer/registry.ts

```diff
@@ -213,6 +215,20 @@ export const fileTypeRegistry: FileTypeEditor[] = [
     subType: "zxbas",
     icon: "@file-zxbas"
   },
+  {
```

Comment:
I think it is nice to have an ability to edit plain text files sometimes. A basic support for now should be enough.